cypress/e2e/utils/index.ts

```diff
@@ -226,6 +226,16 @@ function getAssertionTest(
     scrollSelectorIntoView(selectorStr);
   }
   if (Array.isArray(shouldArgs.value)) {
+    if (shouldArgs.chainer === "be.visible.and.contain") {
+      return cy
+        .get(selectorStr, opts)
+        .should("be.visible")
+        .should($el => {
+          shouldArgs.value.forEach((v: string) =>
+            expect($el).to.contain(v, message),
+          );
+        });
+    }
```

Comment:
Can do something like this
```suggestion
      return cy
        .get(selectorStr, opts)
        .should("be.visible")
        .should($el => {
          shouldArgs.value.forEach((v: string) => {
            if (shouldArgs.chainer === "be.visible.and.contain") {
              expect($el).to.contain(v, message)
            } else if (shouldArgs.chainer === "has.attr") {
              expect($el).has.attr(v, message)
            } else {
              throw new Error(`Chainer ${shouldArgs.chainer} not supported with an array of values`)
             }
          });
        });
```